ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+Please describe the issue you are encountering in Kura:
+
```

Comment:
We can add an initial comment saying:
"Fill the following for only in case you are reporting a **bug**. Provide a textual description otherwise."